SimDataFormats/TrackerDigiSimLink/src/classes_def.xml

```diff
@@ -35,4 +35,18 @@
  -->
  <class name="edm::Wrapper< StripCompactDigiSimLinks >"  />
 
+  <class name="PixelSimHitAddExtraInfo" ClassVersion="4">
+   <field name="chan"     mapping="blob" />
+   <version ClassVersion="3" checksum="2885258605"/>
+   <version ClassVersion="4" checksum="769431277"/>
```

Comment:
Since this class is just being introduced to the release, there is no need to have two versions. I think you can just take the checksum of version 4 and assign it version number 3.